src/tests/Microsoft.PowerFx.Core.Tests/Microsoft.PowerFx.Core.Tests.csproj

```diff
@@ -47,6 +47,11 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="ExpressionTestCases\DecimalBoot.txt" />
+    <None Remove="ExpressionTestCases\FloatLarge.txt" />
+  </ItemGroup>
   
```

Comment:
Not needed. This was added automatically by VS but should be left out

Reply:
Yeah, wasn't sure how the csproj updates worked.  Removed.